### ISSUE TITLE
Missing include in GetDensityRatio.hpp

### ISSUE THREAD

**Issue by PrometheusPi** [ComputationalRadiationPhysics/picongpu#1757]
I am using [`GetDensityRatio.hpp`](https://github.com/ComputationalRadiationPhysics/picongpu/blob/dev/src/picongpu/include/particles/traits/GetDensityRatio.hpp) on a taurus simulation (bases on 0.2.2) and by including this file in `particleConfig.param` I get the following compile error:
```
.../libraries/boost/1.60.0-gnu5.3/include/boost/mpl/if.hpp(63): error: name followed by "::" must be a class or namespace name
          detected during:
            instantiation of class "boost::mpl::if_<T1, T2, T3> [with T1=<error-type>, T2=picongpu::placeholder_definition58::DensityRatioElectrons, T3=picongpu::traits::detail::placeholder_definition47::DefaultDensityRatio]" 
.../picongpu/src/picongpu/include/particles/traits/GetDensityRatio.hpp(61): here
```

How can I solve this issue and use `GetDensityRatio.hpp`?

EDIT: 
caused by missing include in GetDensityRatio.hpp. The above error was not the first error but just a followup error.

**Comment by n01r** [comment]
That was not the full extent of the error message. The part we see here shows that `T1=<error-type>` meaning that it does not understand the first template argument. A few lines above there it says `HasFlag` *is not a template* and that is probably where the error lies.

**Comment by PrometheusPi** [comment]
This was due to a missing include in `GetDensityRatio.hpp` - fixed by #1759.
